requirements.in

```diff
@@ -5,4 +5,7 @@ jax
 jaxlib
 portpicker
 numpy
-seqio
\ No newline at end of file
+seqio
+nltk
+evaluate
+rouge-score
```

Comment:
We can move benchmarks requirements to a separate `requirements.in` file under /benchmarks

Reply:
Thanks, done.